Source/ALSV4_CPP/Public/Components/ALSMantleComponent.h

```diff
@@ -101,4 +104,6 @@ class ALSV4_CPP_API UALSMantleComponent : public UActorComponent
 private:
 	UPROPERTY()
 	AALSBaseCharacter* OwnerCharacter;
+
+	UALSDebugComponent* m_debugComponent = nullptr;
```

Comment:
Please use camel case for variable names. `DebugComponent` should be fine.

Reply:
Changed